src/Sarif.WorkItems/StringSetConverter.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.CodeAnalysis.Sarif.WorkItems
+{
+    public class StringSetConverter : JsonConverter
```

Comment:
>StringSetConverter [](start = 17, length = 18)

I can't seem to find this used anywhere. Maybe intended for later?

Reply:
A left-over class from an investigation I did around the current JSON serialization functionality in PropertiesDictionary. Now deleted.

---
In reply to: [381443881](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381443881) [](ancestors = 381443881)